### ISSUE TITLE
Changing render detail crashes on Windows

### ISSUE THREAD

**Issue by miloush** [shooshx/happysolver#3]
Changing anything on the View > Options > Solution Display tab and clicking OK will make the application crash on Windows 10 64-bit.

**Comment by shooshx** [comment]
Are you using the desktop version? That has not been maintained for along time and I'm not going to try to fix it.
The latest version is the WebGL version:
http://shooshx.github.io/happysolver/emscripten/index.html
That should have most of the features as the desktop version and with a much better UI

**Comment by miloush** [comment]
The web version has a simple user interface, you cannot see the pieces in the selection, you cannot even select the individual pieces - only color sets, there is no way to configure how solutions are searched and how many to return, how to treat pieces with texture on one side, you cannot save or load anything, or see the step by step...

It definitely does not have most features of the desktop version and the UI is very basic compared to the desktop version... or am I missing something?

**Comment by shooshx** [comment]
- You can select individual pieces if you press the name of the individual color set, for instance open "Little Genius" and press on the word "Nature". A panel with checkbox pops up where you can select which pieces to include.
- The default behavior for treating textured pieces is to have the texture on the outside. You can construct the mirrored shape if you want to have them on the inside.
- If you want more solutions than those that are returned, just keep pressing the cogs button again and again until you find something you like.
- You can press the Link button at the bottom right corner you get a URL that "saves" the solution you're currently looking at. you can paste that into your browser to "load" it.

If anything these is not to your liking, if you're missing any other feature, or if you want the desktop version resurrected and the issues there fixed, feel free to fork the project, send a PR with your code and I will review it.

Also, bear in mind that this is an open source project that I've been working on in my free time. You have not payed for this product and I don't owe you anything. Keeping this entitled tone is the best way for you to go nowhere fast.

**Comment by miloush** [comment]
Thank you for the tip on clicking on the color sets.

It was not my intention to suggest that you owe anything to anyone, I am sorry if I gave you that impression. I only tried to list the reasons why your claim that the web version is better is not true. You obviously wrote the desktop version too, so credit goes to you either way.

Also note that according to bottom section of https://www.happycube.com/educational/, I cannot actually modify the code myself. And since your software is endorsed by the company, whether any of my money went to its development is unclear at best.
